MovingBlocks/Terasology
# Add a RunsAfter annotation to allow ordering of rasterizers

### Contains

Adds a `RunsAfter` annotation that indicates one rasterizer must be run after another.

### How to test

Add the annotation to a rasterizer. E.g.
```
@RegisterPlugin
@RunsAfter(CaveRasterizer.class)
public class SomeRasterizer implements WorldRasterizerPlugin {
```
During world generation you can place a breakpoint at `WorldBuilder` line 108. When it hits the breakpoint inspect the `orderedRasterizers` list and confirm that `SomeRasterizer` is after `CaveRasterizer`.
